.core-coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+omit =
+    setup.py
+    tests/*
+    sample_factory/runner/runs/*
+    sample_factory/huggingface/*
```

Comment:
I think we can keep huggingface and PBT. We should only remove folders that we can't test in CI such as doom or isaacgym. But PbT and huggingface we can write tests for. So coverage below 100% should reflect that! :)

Reply:
OK, got it.